src/operations/UpdateContext.ts

```diff
@@ -1,11 +1,17 @@
-import { Operation, createContext } from "effection";
+import { Operation, createContext, lift } from "effection";
 import { LoaderState } from "../hooks/useLoader";
 
-type Update = (state: LoaderState<any>) => Operation<void>;
+type Update<T> = (state: LoaderState<T>) => Operation<void>;
 
-export const UpdateContext = createContext<Update>("update");
+export const UpdateContext = createContext<Update<unknown>>("update");
 
-export function* update<T>(state: LoaderState<T>): Operation<void> {
-  let setState = yield* UpdateContext;
-  yield* setState(state);
+export const update: Update<unknown> = function* update(value) {
+  const setState = yield* UpdateContext;
+  yield* setState(value);
 }
```

Comment:
I would avoid using custom types here and keep it to `Operation<T>`. That way, the language server will in all cases type it as `(state: LoaderStace<T>) => Operation<void>` which is meaningful to the eye, and not `Update<T>` which isn't really.

Reply:
I removed the Update type by using typeof in a few places. Take a look